Poke/src/pages/add-user.tsx

```diff
@@ -59,8 +67,7 @@ export const AddUser: NavigationFunctionComponent<Props> = (props: NavigationCom
     } else if (dateValidator(birthDate)) {
       Alert.alert(dateError);
     } else {
-      Alert.alert('Nice');
-      Navigation.pop(props.componentId);
+      await createUser({variables: {name: name, email: email, phone: phone, birthDate: birthDate}});
```

Comment:
```suggestion
      await createUser({ variables: { name, email, phone, birthDate } });
```

Quando nome da variável coincide com o nome do campo, podemos omitir e deixar assim.